provider/provider.go

```diff
@@ -13,6 +13,7 @@ package provider
 import (
 	"context"
 	"fmt"
+	"github.com/cloudera/terraform-provider-cdp/resources/dw/virtualwarehouse/impala"
```

Comment:
please run `make fix` to fix the import order

Reply:
sure, thanks for the review